packages/react-aria-components/src/FileTrigger.tsx

```diff
@@ -39,11 +39,11 @@ export interface FileTriggerProps {
   /**
    * Enables the selection of directories instead of individual files.
    */
-  directory?: boolean
+  acceptDirectory?: boolean
```

Comment:
did we determine what happens if we specify both allowsMultiple and directory at the same time?

Reply:
yeah i tried it and you can't select multiple directories

Reply:
should it be plural e.g. `acceptDirectories`?

I was also wondering if it should be combined with `acceptedFIleTypes` like we do for `acceptedDragTypes`: https://react-spectrum.adobe.com/react-aria/Table.html#directories

Reply:
we could make it plural since safari supports it but since other browsers don't im a little hestitant

i mentioned this in the little blurb above but i don't think we should combine it with `acceptedFileTypes` because the file input doesn't seem to respect accepted file types if you allow directories (at least in Chrome, i haven't tried it with other browsers). even if you define what your accepted file types are, those will be ignored when you upload a directory. i don't know, for me it just feels like since they don't work together that it'd be best to keep them separate.